.github/CODEOWNERS

```diff
@@ -50,7 +50,9 @@ licenses           @emaxerrno @dswang
 /src/v/config/node_config.h                         @redpanda-data/documentation
 /src/v/kafka/client/configuration.cc                @redpanda-data/documentation
 /src/v/kafka/client/configuration.h                 @redpanda-data/documentation
+/src/v/pandaproxy/api/api-doc                       @redpanda-data/documentation
```

Comment:
there is also `src/v/pandaproxy/admin/api-doc` does that need to be added?

Reply:
Looks like [it's been dead code](https://github.com/redpanda-data/redpanda/pull/682) for 3 years.